### ISSUE TITLE
Difference in node rotation behaviour of the simulator and a browser

### ISSUE THREAD

**Issue by arthurzhukovski** [cocos/cocos-engine#7480]
### Creator version?（版本号）
2.4.0

### Affected platform?（受影响的平台）

Web (tested on Chrome & Firefox on Windows, Safari on iOS)
### How to reproduce?（如何重现）
Rotation of a node around X and Y axes works in simulator but doesn't work in browser unless the node is being explicitly converted to 3D. 
A scipt, that is attached to a regular node with a sprite, contains the following snippet:
```
let angle = 180;
let rotationVec = cc.v3(0, angle, 0);
cc.tween(this.node).by(1, {eulerAngles: rotationVec}).start();
```
It **works perfectly** fine when being run in the **simulator**: the node smoothly rotates around the Y-axis:

![simulator without 3d enabled](https://user-images.githubusercontent.com/15804937/95332677-6e253600-08b4-11eb-9b05-32c6fa2f6791.gif)

But in **bowser** it **doesn't work** - no rotation is being performed: 

![browser without 3d enabled](https://user-images.githubusercontent.com/15804937/95332680-6ebdcc80-08b4-11eb-8deb-2c5392e404b5.gif)

It doesn't rotate around X-axis as well, when the value of  ```rotationVec``` is changed to ```cc.v3(angle, 0, 0)```.
If the value of  ```rotationVec``` is changed to ```cc.v3(0, 0, angle)```, the node rotates around Z-axis, but without animation/easing. 

### Solution
The node must be explicitly converted to 3D:
![solution](https://user-images.githubusercontent.com/15804937/95331036-11288080-08b2-11eb-969f-f96f83bd3572.png)

Now both the simulator and the browser behave identically:
![browser with 3d enabled](https://user-images.githubusercontent.com/15804937/95332681-6f566300-08b4-11eb-8967-779a4259714f.gif)

**Comment by SantyWang** [comment]
Thanks, we will check it out

**Comment by zhefengzhang** [comment]
This bug will be fixed in version 3.0.

**Comment by yeshao2069** [comment]
Thanks for the feedback, this problem has been fixed in version 3.0, you can upgrade to version 3.x and try again.
[testRotate.zip](https://github.com/cocos-creator/engine/files/6837758/testRotate.zip)
